jovian/jovian_nb_ext/main.js

```diff
@@ -28,6 +28,8 @@ define([
         const jvnLog = data => {
           resolve(data.content.text.trim());
         };
+        /* Saves checkpoints before committing*/
+        Jupyter.notebook.save_checkpoint();
```

Comment:
Better give a 1-2 second delay before continuing, because this is an asynchronous action.

Reply:
@PrajwalPrashanth did you get a chance to look into this?

Reply:
can you see if you wait for this event: https://github.com/jupyter/notebook/blob/8b21329deb9dd04271b12e3d2790c5be9f1fd51e/notebook/static/notebook/js/notebook.js#L2867

Reply:
![image](https://user-images.githubusercontent.com/24864161/66217419-e1fe9280-e6e4-11e9-96ad-ab9cfc831f43.png)

Yes we can do that before committing, shall we do it for the library as it won't require explicitly doing this in the extension.

Reply:
But if i commit for that event then it commit for every autosave right?

Reply:
we can do it for extension for now, can do for library in a separate PR

Reply:
Done.